experiment/autobumper/bumper/bumper.go

```diff
@@ -139,6 +185,16 @@ type GitAuthorOptions struct {
 	GitEmail string
 }
 
+// UnmarshallWithDefaults unmarshalls yaml config file using default options for unspecified fields
+func UnmarshallWithDefaults(data []byte) (Options, error) {
```

Comment:
Please unexport this and export `defaultOptions` instead. Ppl may or may not want to use a configfile for this, but they will definitely want to get the defaults, as half of it is essentially constant

/ok-to-test

Reply:
I left this exported so that I can use it in main.go, but I exported DefaultOptions as well now.